assets/js/customizer.js

```diff
@@ -424,7 +449,6 @@
 					this.initSortable(searchList, {
 					    group: {
 					        name: 'nested',
-					        pull: 'clone',
```

Comment:
@redundans Is it a bad idea to remove this? Not sure why it's there but since it's specific (i.e. this option does not need to be set) it might have a good reason I don't know about? :)

Reply:
I don't see any problems with this when I look.